src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+  determine the look and feel of your application window and to decide which default browser controls—potentially none of them—should be shown.
+{% endDetails %}
+
+## Snappy animations
+
+In-app animations are snappy and smooth in Podcasts. For example, when I open the "Episode Notes" drawer on the right, it elegantly slides in. When I remove one episode from my downloads, the remaining episodes float up and consume the screen real estate that was freed by the deleted episode.
+
+<figure class="w-figure">
+  <img src="./image14.png" alt="The Podcasts app with the 'Episode Notes' drawer expanded." width="600">
+  <figcaption class="w-figcaption">In-app animations like when opening a drawer are snappy.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  Performant animations on the web are certainly possible if you take into account a number of best practices
+  outlined in the article <a href="https://developers.google.com/web/fundamentals/design-and-ux/animations/animations-and-performance">Animations and Performance</a>.
+  Scroll animations as commonly seen in paginated content or media carousels can be massively improved by using the <a href="https://developers.google.com/web/updates/2018/07/css-scroll-snap">CSS Scroll Snap</a> feature.
+  For full control, you can use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API">Web Animations API</a>.
+{% endDetails %}
+
+## Content surfaced outside of app
+
+The Podcasts app on iOS can surface content in other locations than the actual application, for example, in the system's Widgets view, or in the form of a Siri Suggestion. Having proactive, usage-based calls-to-action that just require a tap to interact with can greatly increase the re-engagement rate of an app like Podcasts.
+
+<figure class="w-figure">
+  <img src="./image6.png" alt="iOS Widget view showing the Podcasts app suggesting a new episode of a podcast." width="300">
+  <figcaption class="w-figcaption">App content is surfaced outside of the main Podcasts app.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/content-indexing-api/">Content Index API</a> allows your application
+  to tell the browser which content of the PWA is available offline.
+  This allows the browser to surface this content outside of the main app.
+  By marking up interesting content in your app as suitable for <a href="https://developers.google.com/search/docs/data-types/speakable">speakable</a>
+  audio playback and by using <a href="https://developers.google.com/search/docs/guides/search-gallery">structured markup</a> in general,
+  you can help search engines and virtual assistants like the Google Assistant present your offerings in an ideal light.
+{% endDetails %}
```

Comment:
Can I get my PWA notification to show up on my iOS device? This seems misleading to me.

Reply:
Conditions apply, yes. Let me find a way to address the browser compatibility questions.